src/components/DisplayText/tests/DisplayText.test.tsx

```diff
@@ -29,11 +29,4 @@ describe('<DisplayText />', () => {
     );
     expect(displayText.find('p')).toHaveLength(1);
   });
-
-  it('renders the specified size', () => {
```

Comment:
We shouldn't expect on the base components props since it doesn't provide value. This test should be handled with visual regression like percy